app/services/diagnostics/move_inspector.rb

```diff
@@ -36,9 +36,9 @@ def generate
         -----------
       ENDMOVEEVENTS
 
-      if move.move_events.any?
+      if move.events.any?
```

Comment:
When I come to remove dual writing I will replace these with the generic events relationship